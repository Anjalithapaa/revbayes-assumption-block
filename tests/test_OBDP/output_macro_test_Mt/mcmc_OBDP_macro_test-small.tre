#NEXUS

	Begin taxa;
	Dimensions ntax=10;
	Taxlabels
		f0
		f1
		f2
		f3
		f4
		f50
		f51
		f52
		f53
		f54
		;
End;

Begin trees;
tree TREE1 = [&R]((((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11,posterior=0.625000,age_95%_HPD={2.75775,2.75775}]:1.190248)f52[&index=12,posterior=0.625000,age_95%_HPD={3.948,3.948},sampled_ancestor=0.750000]:0.416445,(f4[&index=4]:3.803000)f51[&index=13,posterior=0.562500,age_95%_HPD={3.803,3.803},sampled_ancestor=0.562500]:0.561445)[&index=14,posterior=0.500000,age_95%_HPD={4.36444,4.36444}]:0.263803,f1[&index=2]:4.628248)[&index=15,posterior=0.562500,age_95%_HPD={4.62825,4.62825}]:0.304965,((f3[&index=6]:3.114000)f50[&index=16,posterior=0.562500,age_95%_HPD={3.114,3.114},sampled_ancestor=1.000000]:1.459588,f53[&index=9,age_95%_HPD={4.2,4.2}]:0.373588)[&index=17,posterior=0.562500,age_95%_HPD={4.57359,4.57359}]:0.359625)[&index=18,posterior=1.000000,age_95%_HPD={4.65609,4.9728}]:0.227787)f54[&index=19,posterior=0.687500,age_95%_HPD={5.161,5.161},sampled_ancestor=0.687500]:0.000000;
End;
