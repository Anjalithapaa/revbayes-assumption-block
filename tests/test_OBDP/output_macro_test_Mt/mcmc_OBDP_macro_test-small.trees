Iteration	Posterior	Likelihood	Prior	obd_tree
0	-53875.28	-53797.74	-77.53997	(f54[&index=10]:0.066660,((f53[&index=9]:0.700067,(f2[&index=8]:4.282443,((f0[&index=7]:2.757752,f3[&index=6]:2.757752)[&index=11]:1.441970,(f50[&index=5]:0.280970,f4[&index=4]:3.394970)[&index=12]:0.804752)[&index=13]:0.082721)[&index=14]:0.617625)[&index=15]:0.298206,(f52[&index=3]:0.475853,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.059407)[&index=17]:0.774421)[&index=18]:0.029387)[&index=19]:0.000000;
5	-53875.18	-53798.32	-76.86479	(f54[&index=10]:0.170584,((f53[&index=9]:0.424891,(f2[&index=8]:4.282443,((f0[&index=7]:2.757752,f3[&index=6]:2.757752)[&index=11]:1.441970,(f50[&index=5]:0.280970,f4[&index=4]:3.394970)[&index=12]:0.804752)[&index=13]:0.082721)[&index=14]:0.342448)[&index=15]:0.573382,(f52[&index=3]:0.680248,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.263803)[&index=17]:0.570025)[&index=18]:0.133310)[&index=19]:0.000000;
10	-53668.86	-53591.4	-77.46081	(f54[&index=10]:0.045005,((f53[&index=9]:0.424891,((f0[&index=7]:2.757752,f3[&index=6]:2.757752)[&index=11]:1.408494,(f50[&index=5]:0.280970,f4[&index=4]:3.394970)[&index=12]:0.771277)[&index=13]:0.458645)[&index=15]:0.573382,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.263803)[&index=17]:0.570025)[&index=18]:0.007731)[&index=19]:0.000000;
15	-53606.94	-53535.35	-71.59196	(f54[&index=10]:0.045005,((f53[&index=9]:0.424891,(f3[&index=6]:4.166246,((f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:0.356248)f50[&index=12]:1.052246)[&index=13]:0.458645)[&index=15]:0.507262,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.263803)[&index=17]:0.503905)[&index=18]:0.073851)[&index=19]:0.000000;
20	-53399.42	-53327.83	-71.59196	(f54[&index=10]:0.045005,(f53[&index=9]:0.932154,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,(f1[&index=2]:4.624891,((f3[&index=6]:4.166246,((f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:0.356248)f50[&index=12]:1.052246)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357)[&index=17]:0.503905)[&index=18]:0.073851)[&index=19]:0.000000;
25	-53398.88	-53327.36	-71.5129	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,(((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.342448,((f3[&index=6]:4.166246,((f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:0.356248)f50[&index=12]:1.052246)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;
30	-53399.55	-53328.04	-71.5129	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,((f1[&index=2]:3.114000)f50[&index=12]:1.510891,((f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;
35	-53399.46	-53327.95	-71.5129	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,(((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.342448,((f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;
40	-53399.79	-53328.28	-71.5129	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,((((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.082003,f51[&index=1]:0.561445)[&index=16]:0.260446,(f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.458645)[&index=15]:0.003357,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;
45	-53393.49	-53328.07	-65.42012	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,((((f2[&index=8]:3.948000)f52[&index=14]:0.416445,f51[&index=1]:0.561445)[&index=16]:0.260446,(f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.458645)[&index=15]:0.003357,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;
50	-53359.01	-53300.04	-58.96932	((f53[&index=9]:0.772803,((((f2[&index=8]:3.948000)f52[&index=14]:0.416445,(f51[&index=1]:0.363246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.198199)[&index=16]:0.209143,f3[&index=6]:4.573588)[&index=15]:0.054660,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
55	-51089.69	-51030.72	-58.96932	(((f53[&index=9]:0.373588,f3[&index=6]:4.573588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f51[&index=1]:0.363246,f4[&index=4]:4.166246)[&index=13]:0.198199)[&index=16]:0.263803,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
60	-51084.1	-51031.03	-53.06681	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
65	-51084.1	-51031.03	-53.06681	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
70	-51084.1	-51031.03	-53.06681	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
75	-51084.1	-51031.03	-53.06681	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
80	-51084.13	-51031.03	-53.09678	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
85	-51084.13	-51031.03	-53.09678	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
90	-50344.62	-50291.52	-53.09678	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((f4[&index=4]:3.803000)f51[&index=13]:0.825248,(((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,f1[&index=2]:4.364445)[&index=16]:0.263803)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;
95	-50338.5	-50285.57	-52.9261	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.082498,((f4[&index=4]:3.803000)f51[&index=13]:0.825248,(((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,f1[&index=2]:4.364445)[&index=16]:0.263803)[&index=17]:0.027838)[&index=18]:0.504914)f54[&index=19]:0.000000;
100	-50338.5	-50285.57	-52.9261	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.082498,((f4[&index=4]:3.803000)f51[&index=13]:0.825248,(((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,f1[&index=2]:4.364445)[&index=16]:0.263803)[&index=17]:0.027838)[&index=18]:0.504914)f54[&index=19]:0.000000;
