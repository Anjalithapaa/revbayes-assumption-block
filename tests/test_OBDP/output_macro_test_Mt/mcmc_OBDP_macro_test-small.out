Iteration	alpha_morpho	alpha_seq	branch_rates	clock_morpho	er	lambda	mu	obd_tree	omega	psi	rho	sf	start_time
0	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.066660,((f53[&index=9]:0.700067,(f2[&index=8]:4.282443,((f0[&index=7]:2.757752,f3[&index=6]:2.757752)[&index=11]:1.441970,(f50[&index=5]:0.280970,f4[&index=4]:3.394970)[&index=12]:0.804752)[&index=13]:0.082721)[&index=14]:0.617625)[&index=15]:0.298206,(f52[&index=3]:0.475853,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.059407)[&index=17]:0.774421)[&index=18]:0.029387)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.48687
5	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.170584,((f53[&index=9]:0.424891,(f2[&index=8]:4.282443,((f0[&index=7]:2.757752,f3[&index=6]:2.757752)[&index=11]:1.441970,(f50[&index=5]:0.280970,f4[&index=4]:3.394970)[&index=12]:0.804752)[&index=13]:0.082721)[&index=14]:0.342448)[&index=15]:0.573382,(f52[&index=3]:0.680248,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.263803)[&index=17]:0.570025)[&index=18]:0.133310)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.48687
10	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.045005,((f53[&index=9]:0.424891,((f0[&index=7]:2.757752,f3[&index=6]:2.757752)[&index=11]:1.408494,(f50[&index=5]:0.280970,f4[&index=4]:3.394970)[&index=12]:0.771277)[&index=13]:0.458645)[&index=15]:0.573382,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.263803)[&index=17]:0.570025)[&index=18]:0.007731)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.46602
15	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.045005,((f53[&index=9]:0.424891,(f3[&index=6]:4.166246,((f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:0.356248)f50[&index=12]:1.052246)[&index=13]:0.458645)[&index=15]:0.507262,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,(f1[&index=2]:4.364445,f51[&index=1]:0.561445)[&index=16]:0.263803)[&index=17]:0.503905)[&index=18]:0.073851)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.46602
20	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.045005,(f53[&index=9]:0.932154,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,(f1[&index=2]:4.624891,((f3[&index=6]:4.166246,((f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:0.356248)f50[&index=12]:1.052246)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357)[&index=17]:0.503905)[&index=18]:0.073851)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.46602
25	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,(((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.342448,((f3[&index=6]:4.166246,((f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:0.356248)f50[&index=12]:1.052246)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
30	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.345806,((f1[&index=2]:3.114000)f50[&index=12]:1.510891,((f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
35	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,(((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.342448,((f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.198199,f51[&index=1]:0.561445)[&index=16]:0.260446)[&index=15]:0.003357,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
40	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,((((f2[&index=8]:4.282443,f52[&index=3]:0.334443)[&index=14]:0.082003,f51[&index=1]:0.561445)[&index=16]:0.260446,(f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.458645)[&index=15]:0.003357,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
45	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(f54[&index=10]:0.017924,(f53[&index=9]:0.772803,((((f2[&index=8]:3.948000)f52[&index=14]:0.416445,f51[&index=1]:0.561445)[&index=16]:0.260446,(f3[&index=6]:4.166246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.458645)[&index=15]:0.003357,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.206121)[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
50	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	((f53[&index=9]:0.772803,((((f2[&index=8]:3.948000)f52[&index=14]:0.416445,(f51[&index=1]:0.363246,(f0[&index=7]:2.757752,f4[&index=4]:2.757752)[&index=11]:1.408494)[&index=13]:0.198199)[&index=16]:0.209143,f3[&index=6]:4.573588)[&index=15]:0.054660,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
55	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,f3[&index=6]:4.573588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f51[&index=1]:0.363246,f4[&index=4]:4.166246)[&index=13]:0.198199)[&index=16]:0.263803,(f1[&index=2]:3.114000)f50[&index=12]:1.514248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
60	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
65	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
70	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
75	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.505966
80	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.561674
85	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,(f4[&index=4]:3.803000)f51[&index=13]:0.561445)[&index=16]:0.263803,f1[&index=2]:4.628248)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.561674
90	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.399214,((f4[&index=4]:3.803000)f51[&index=13]:0.825248,(((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,f1[&index=2]:4.364445)[&index=16]:0.263803)[&index=17]:0.344554)[&index=18]:0.188197)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.561674
95	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.082498,((f4[&index=4]:3.803000)f51[&index=13]:0.825248,(((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,f1[&index=2]:4.364445)[&index=16]:0.263803)[&index=17]:0.027838)[&index=18]:0.504914)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.561674
100	5.632603	1.729891	0.09028198	0.1416907	[0.177935,0.116682,0.128997,0.0796538,0.39152,0.105213]	0.007298359	0.011636	(((f53[&index=9]:0.373588,(f3[&index=6]:3.114000)f50[&index=12]:1.459588)[&index=15]:0.082498,((f4[&index=4]:3.803000)f51[&index=13]:0.825248,(((f0[&index=7]:2.757752,f2[&index=8]:2.757752)[&index=11]:1.190248)f52[&index=14]:0.416445,f1[&index=2]:4.364445)[&index=16]:0.263803)[&index=17]:0.027838)[&index=18]:0.504914)f54[&index=19]:0.000000;	0.406959	0.1150825	0.1839188	[0.208893,0.0808167,0.554983,0.155307]	5.561674
